util/encryption.go

```diff
@@ -26,7 +26,9 @@ func init() {
 // variable again. Useful for testing purposes outside the "util" package.
 func InitializeEncryption() {
 	key = os.Getenv("ENCRYPTION_KEY")
-
+	if key == "" {
+		_ = os.Setenv("ENCRYPTION_KEY", "YWFhYWFhYWFhYWFhYWFhYQ")
+	}
```

Comment:
I believe this is not working as we need

if key is empty we set there some default value - but the value is set only into environment variables, the "key" variable stays empty 
+
maybe  we can use here some nice private function to set default value, something like
```
if key == "" {
    key = setDefaultEncryptionKey()
}
```